config/update.sh

```diff
@@ -0,0 +1,4 @@
+# Get previous ETAG
```

Comment:
Don't know how this is intended to be used for, but I suggest the following improvements
1) Add a shebang, so that we know which shell this is intended to be parsed by (even if it si to be sourced, that is helpful)

2) Add a brief (one or two lines) of comments at the top stating the usage of the script.  I see that $1 is being used further down, but it's hard to determine what that's ment to be from just staring at the code.   Bash is concise, so some level of documentation is necessary

3) If. this is bash, use $(..) instead of `...` to get results from sub shells

4) The ETAG=... line is fair complex, so an one (or up to three)-line comment above it would help
     comprehension a lot.

5) Not necessary, but a practice I generally try to follow, so I'll mention it for you:  You use several subcommands (node, grep, cut, sed) that are usually, but not always installed.   I find it to be a good practice to have list of necessary subcommands, and then check if they can be found, and fail early if not.   In this particular script I actually don't know what the result would be, but in general the benefit is that the script fails early, rather than chugging through some process, and perhaps leaving something in an intermediate state, and then failing because a dependency can't be found.  It is much better to fail early with a clear statement about what is missing, and no intermediate state to clean up.

Reply:
1. Thanks, forgot 👍 
2. Added
3. Wrapped the $1 in "$1" though the values are only supposed to be prod and dev and the script is supposed to be run through npm using `npm run deploy:dev` or `npm run deploy:prod` as described in the `README`. I've made a comment about it in the `update.sh` to be more clear about it. Also, ideally, you would also validate the values of $1 so you can't use anything else.
4. I've added a comment describing what it does.
5. I agree, node and npm are already listed as dependencies in the `README`, could probably be clearer, but they are listed under `Environment`. Also agree that it would be great to check what tools are available etc, but it felt like too much work for a not so big task, also each step can be run manually and the script will probably be rewritten at some point. Thus as long as there are enough comments to give comprehensions of what's going it.